### DISCUSSION TITLE
Signed URL Generation for old certificates - cert-service

### DISCUSSION THREAD

**Discussion by ashokreddy1208** [Sunbird-Lern/Community#39] [Documentation]
Hi @reshmi-nair,

Can you please provide more information on below ticket which are part of 5.0 Sunbird-Learn release.

What is the functional impact, deployment change in existing jobs, any migration required etc.,

https://project-sunbird.atlassian.net/browse/LR-108

cc : @anandvarada @kameshbhr @mmanimegalai

**Comment by reshmi-nair** [comment]
@ashokreddy1208 Updated the ticket with relevant information
